src/rocm_docs/rocm_docs_theme/components/left-side-menu.html

```diff
@@ -1,6 +1,6 @@
 {% macro navbar_item(branch) -%}
-    {% if branch in ["develop", "amd-master", "amd-staging", "master"] %}
```

Comment:
I think you want the master branch as well

Reply:
Added back in [2c26f64](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/211/commits/2c26f644f777d403351f556b8762ef54f8f78cfe)